internals/webhooks/updatequota_webhook.go

```diff
@@ -159,6 +160,19 @@ func (a *UpdateQuotaAnnotator) Handle(ctx context.Context, req admission.Request
 		return admission.Denied("Quota Object " + updatingObject.Object.(*danav1.Updatequota).Spec.DestNamespace + " does not exist")
 	}
 
+	//deny update of updatequota object after it's been created(when the status phase is not empty)
+	//check if the current updatequota object in the cluster(if it's present) has status
+	//if the status is not empty - denied the update of the updatequota object
+	currentUpdateQuota, err := utils.NewObjectContext(ctx, log, a.Client, client.ObjectKey{Name: req.Name, Namespace: req.Namespace}, &danav1.Updatequota{})
+	if err != nil {
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+	if currentUpdateQuota.IsPresent() {
+		if !reflect.ValueOf(currentUpdateQuota.Object.(*danav1.Updatequota).Status).IsZero() {
+			return admission.Denied("Quota Object " + updatingObject.Object.(*danav1.Updatequota).ObjectMeta.Name + " already finished ")
+		}
+	}
+
```

Comment:
You can probably mirror the logic that already exists on the Migration Hierarchy webhook [here](https://github.com/dana-team/hns/blob/main/internals/webhooks/migrationhierarchy_webhook.go#L152-L164). I would change the returned message though (in both cases) to be something along the lines of: `"It is forbidden to update an object of type <type_name>"`